### DISCUSSION TITLE
Is there a list of keyboard shortcuts?

### DISCUSSION THREAD

**Discussion by syntaxterror9** [zanllp/infinite-image-browsing#461] [Q&A] [answered]
I'm looking for the keyboard shortcut for tagging/liking an image, which key should be pressed?

Is there a list for the complete set of keyboard shortcuts?

Thanks!

**Comment by zanllp** [comment, marked as answer]
This extension doesn't have a default shortcut key. You need to manually set it according to your preference.

https://github.com/zanllp/sd-webui-infinite-image-browsing/issues/159
